lib/dateTime.js

```diff
@@ -152,6 +186,37 @@ function processPattern(pattern, replacements) {
 
 }
 
+export function convertUTCToLocalDateTime(date) {
+	const timezone = getDocumentLocaleSettings().timezone.identifier;
```

Comment:
Can we make sure that if `timezone.identifier` is blank for whatever reason that the conversion just does nothing?